@uportal/eyebrow-user-info/package.json

```diff
@@ -1,14 +1,18 @@
 {
   "name": "@uportal/eyebrow-user-info",
-  "version": "1.36.0",
+  "version": "1.37.0",
   "scripts": {
     "start": "vue-cli-service serve",
     "prebuild": "babel node_modules/@vue/web-component-wrapper/dist/vue-wc-wrapper.js -o node_modules/@vue/web-component-wrapper/dist/vue-wc-wrapper.js",
     "build": "vue-cli-service build --name eyebrow-user-info --target wc \"./src/components/EyebrowUserInfo.vue\"",
     "lint": "vue-cli-service lint"
   },
+  "engines": {
+    "node": "^14.0.0",
+    "npm": "^6.0.0"
+  },
```

Comment:
Interesting to see this, are there issues installing dependencies with Node 16? (Ideally we want to support 16/current long term support release)
CI seemed to be happy https://github.com/uPortal-Project/uPortal-web-components/pull/831 :thinking:
But I haven't tested manually yet.